examples/desk_security_guard.py

```diff
@@ -48,8 +48,8 @@
 class TwitterStreamToAppCommunication:
     '''Class for messaging to/from SecurityGuardStreamListener
 
-    Tweepy doesn't support asyncio, so we have to run the SecurityGuardStreamListener
-    stream in its own thread. We limit the communication between the rest of the
+    Tweepy doesn't support asyncio, so it has to run the SecurityGuardStreamListener
+    stream in its own thread. Communication is limited between the rest of the
```

Comment:
Suggest "Communication is deliberately limited" if you wish to reword this, it needs to convey that this is a deliberate choice rather than a limitation.